queries.sql

```diff
@@ -16,25 +16,41 @@ SELECT name,escape_attempts FROM animals WHERE weight_kg>10.5;
   /* all animals with a weight between 10.4kg and 17.3kg */
    SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
 
+     
 
-
-
-
+    BEGIN TRANSACTION;
+    update animals set species = 'unspecified';
+    ROLLBACK;
+    
+    BEGIN TRANSACTION;
     update animals set species = 'digimon' where name LIKE '%mon%';
     update animals set species = 'pokemon' where species='';
+    select species from animals;
+    COMMIT;
+    select species from animals;
+
+    BEGIN TRANSACTION;
+    delete from animals;
+    select * from animals;
+    ROLLBACK;
+    select * from animals;
+
+    BEGIN TRANSACTION;
     delete from animals where date_of_birth > '2022-01-01';
-    /*  savepoint s1; */
+    savepoint s1;
```

Comment:
- [Optional] Good job creating a save point @MayPyone, But I would suggest having a meaning full name while doing so. This helps to make your code more readable and communicate what the save point is for.